chalicelib/dcc/transformer_mapping.json

```diff
@@ -65,103 +46,85 @@
         "dss_field": "sha1"
       },
       {
-        "source": "metadata.json",
-        "index_field": "experimentalStrategy",
-        "dss_field": "submitter_experimental_design"
-      },
-      {
-        "source": "metadata.json",
-        "dss_field": "bundle_uuid",
+        "from_file_data": true,
+        "dss_field": "uuid",
         "index_field": "download_id"
       },
       {
         "source": "metadata.json",
-        "index_field": "donor",
```

Comment:
Let's keep "donor" because the dashboard service expects it to be there and I think we will likely be able to get values for this at some point in the future. Let's make it an empty string constant for now.